src/parser-services.ts

```diff
@@ -63,7 +63,9 @@ export function define(rootAST: ESLintProgram): ParserServices {
 
             // If this is the first time, initialize the intermediate event emitter.
             if (emitter == null) {
-                emitters.set(rootAST, (emitter = new EventEmitter()))
+                emitter = new EventEmitter()
+                emitter.setMaxListeners(0)
```

Comment:
This defines unlimited listeners http://nodejs.org/docs/v0.4.7/api/events.html#emitter.setMaxListeners

Reply:
please, where ai find this file? I'm this error later install the plugin eslint-config-vuetify. thanks